Assets/Scripts/BulletManagement/SlipTimeEmitter.cs

```diff
@@ -39,7 +39,7 @@ public class SlipTimeEmitter : EmitterBase, ISlipTimeAdherent
         // Determines if the emitter should aim at the player or not. Note that this will overwrite startAngle.
         public bool aimed = false;
 
-        void Start()
+        private new void Start()
```

Comment:
We should use the `new` keyword here since we're hiding the `Start()` function from the parent class.